### PR TITLE
Clean up Fluent Theme

### PR DESCRIPTION
**Pull request by amwx**
## What does the pull request do?

Went through the FluentTheme files to remove any unused or duplicate resources (only control specific resources, all base colors/brushes were left alone). Also fixed any resources that were misnamed or missing in a couple of the control templates.

`FluentBase[Light/Dark].xaml` and `FluentControlResources[Light/Dark].xaml` had many duplicate resources, so I've marked `FluentBase...` to be removed and put all control resources in `FluentControlResources...`
I've also moved the accent colors to a separate xaml file, which was going to be done in #5374, but made sense to start it now

For now, I've left any changes as comments (and marked 'unused') for now, for easy reverting if any should need/want to be saved.

I automated most of this, but went through control catalogue and everything seems ok for both light/dark theme.

## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Avaloniaui.net with user documentation

### CODE REVIEW

**Comment by maxkatz6** [comment]
> so I've marked FluentBase... to be removed

IIRC initially this file was left to keep UWP resources separated from WinUI resources. But it increases maintenance cost, and I agree to remove it. FYI @danwalmsley

**Comment by maxkatz6** [comment]
@amwx by the way, is it WIP PR or ready to review? Asking because there are a lot of commented code, which we don't really need, and it can be removed.

**Comment by amwx** [comment]
> 
> 
> @amwx by the way, is it WIP PR or ready to review? Asking because there are a lot of commented code, which we don't really need, and it can be removed.

It's ready. I left the comments so if something seems off or a resource needs to be kept, it's easy to get it back. I planned to remove before merge

File: src/Avalonia.Themes.Fluent/FluentDark.xaml [hunk 1]
```diff
@@ -1,9 +1,10 @@
 <Styles xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-       xmlns:sys="clr-namespace:System;assembly=netstandard">  
+       xmlns:sys="clr-namespace:System;assembly=netstandard">
+  <StyleInclude Source="avares://Avalonia.Themes.Fluent/Accents/AccentColors.xaml" />
   <StyleInclude Source="avares://Avalonia.Themes.Fluent/Accents/BaseDark.xaml" />
   <StyleInclude Source="avares://Avalonia.Themes.Fluent/Accents/Base.xaml" />
-  <StyleInclude Source="avares://Avalonia.Themes.Fluent/Accents/FluentBaseDark.xaml" />
+  <!--<StyleInclude Source="avares://Avalonia.Themes.Fluent/Accents/FluentBaseDark.xaml" />-->
```
Comment (maxkatz6) [inline src/Avalonia.Themes.Fluent/FluentDark.xaml:7]: This line can be removed

File: src/Avalonia.Themes.Fluent/FluentLight.xaml [hunk 2]
```diff
@@ -1,9 +1,10 @@
 <Styles xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:sys="clr-namespace:System;assembly=netstandard">
+  <StyleInclude Source="avares://Avalonia.Themes.Fluent/Accents/AccentColors.xaml" />
   <StyleInclude Source="avares://Avalonia.Themes.Fluent/Accents/BaseLight.xaml" />
   <StyleInclude Source="avares://Avalonia.Themes.Fluent/Accents/Base.xaml" />  
-  <StyleInclude Source="avares://Avalonia.Themes.Fluent/Accents/FluentBaseLight.xaml" />
+  <!--<StyleInclude Source="avares://Avalonia.Themes.Fluent/Accents/FluentBaseLight.xaml" />-->
```
Comment (maxkatz6) [inline src/Avalonia.Themes.Fluent/FluentLight.xaml:7]: Same

**Review by maxkatz6 — APPROVED** [review]
LGTM
Also, a bonus https://github.com/AvaloniaUI/Avalonia/pull/5888#issuecomment-835639651

**Review by danwalmsley — APPROVED** [review]

**Review by danwalmsley — APPROVED** [review]

**Pull request merged**